tests/foreman/cli/test_repository.py

```diff
@@ -560,6 +562,42 @@ def test_positive_create_docker_repo_with_name(self, repo_options, repo):
         for key in 'url', 'content-type', 'name':
             assert repo.get(key) == repo_options[key]
 
+    @pytest.mark.tier1
+    def test_positive_create_with_new_organization_and_location(self, default_sat):
+        """Check if error is thrown when creating a Repo with a new Organization and Location.
+
+        :id: 9ea4f2a9-f339-4215-b301-cd39c6b5c474
+
+        :parametrized: no
+
+        :expectedresults: No error is present when Repository is created
+
+        :BZ: 1992967
+
+        :CaseImportance: Critical
+        """
+        new_org = make_org()
+        new_location = make_location()
+        new_product = make_product(
+            {'organization-id': new_org['id'], 'description': 'test_product'}
+        )
+        Org.add_location({'location-id': new_location['id'], 'name': new_org['name']})
+        assert new_location['name'] in Org.info({'id': new_org['id']})['locations']
+        make_repository(
+            {
+                'location-id': new_location['id'],
+                'organization-id': new_org['id'],
+                'product-id': new_product['id'],
+            }
+        )
+
+        result = default_sat.execute(
+            "cat /var/log/foreman/production.log | "
+            "grep \"undefined method `id' for nil:NilClass (NoMethodError)\""
+        )
+        assert result.status == 1
+        assert result.stdout == ""
```

Comment:
Any reason why we are asserting stdout for empty string? If the status is 1, shouldn't that be enough here?